src/components/card/BasicCard.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+    <el-card class="box-card">
+        <div slot="header" class="clearfix">
+            <slot name="title"></slot>
+        </div>
+        <slot name="items"></slot>
+    </el-card>
+</template>
+
+<script>
+export default {
+  name: 'BasicCard'
+}
+</script>
+
+<style lang="scss">
```

Comment:
Perhaps scoping these styles might be a good idea? I imagine lots of different components would have a div with a 'text' or 'item' class. 
If you do want them applied globally then maybe we could put them in a file similar to `element-variables.scss`
```suggestion
<style lang="scss" scoped>
```